src/trace/datadog-tracer.ts

```diff
@@ -0,0 +1,14 @@
+import Tracer from "dd-trace";
+
+const AWS_LAMBDA_FUNCTION_NAME = "AWS_LAMBDA_FUNCTION_NAME";
+
+export function initDatadogTracer() {
+  const functionName = process.env[AWS_LAMBDA_FUNCTION_NAME];
+  const service = functionName ? `${functionName}` : "lambda";
+  Tracer.init({
+    experimental: {
+      exporter: "log",
+    },
+    service,
+  });
+}
```

Comment:
Can we add a debug log for tracer init complete to help make sure tracer is indeed initialized when customers having issues with tracing not working?